### ISSUE TITLE
Only update with specific endpoint documentation

### ISSUE THREAD

**Issue by pingcheng** [knuckleswtf/scribe#267]
- [x] I've read the [documentation](https://scribe.knuckles.wtf/laravel) and I can't find details on how to achieve this.

Is there a way to only update a single endpoint
If not, I would suggest

```
./artisan scrib:generate --only method@SampleApiController
```

**Comment by shalvah** [comment]
No, there isn't, and there likely won't be. What's the use case for this?

**Comment by pingcheng** [comment]
Thanks for the reply.

The main reason is we build the docs on one server and commit the built doc files into the git. so that all developers will have the same copy of API docs. 

Sometimes we will re-seed the DB with random data, and then many "auto-generated" response samples would be updated due to the data in DB is different. However, what we really did is just adding a new endpoint or update an existing single endpoint.

So it could be nice if we can just update a single endpoint (or multiple) docs and not touch the rest

**Comment by shalvah** [comment]
Hmm, I'll think about that.
